<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RechargeWidget</class>
 <widget class="QWidget" name="RechargeWidget">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>充值界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
color:rgb(78, 78, 78，80);
background-color: rgb(247, 235, 183);

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;汉仪文黑-65W&quot;;
color:rgb(54, 54, 54)</string>
       </property>
       <property name="text">
        <string>车牌号:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="vehicleIDLineEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	border-width:2px;
	border-style:solid;
	border-color:gray;
	border-radius:10px;
	padding: 2px 2px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="annualRadioButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>汉仪文黑-85W</family>
          <pointsize>12</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;汉仪文黑-85W&quot;;
color: rgb(54, 54, 54);</string>
        </property>
        <property name="text">
         <string>年卡(365天)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="monthlyRadioButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;汉仪文黑-85W&quot;;
color: rgb(54, 54, 54);</string>
        </property>
        <property name="text">
         <string>月卡(30天)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="quarterlyRadioButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;汉仪文黑-85W&quot;;
color: rgb(54, 54, 54);</string>
        </property>
        <property name="text">
         <string>季卡(90天)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>汉仪文黑-85W</family>
            <pointsize>14</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:!hover{
	font: 14pt  &quot;汉仪文黑-85W&quot;;
	background-color: rgb(255, 255, 255,70);
	
	color: rgb(170, 170, 0);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }
QPushButton:hover{
	font: 14pt  &quot;汉仪文黑-85W&quot;;
	background-color: rgb(123, 117, 69,70);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }
QPushButton:pressed{
	font: 14pt &quot;汉仪文黑-85W&quot;;
	background-color: rgb(123, 117, 69,40);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }</string>
          </property>
          <property name="text">
           <string>充值</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
